config/settings.yaml.example

```diff
@@ -37,6 +37,11 @@
 #:logging:
 #  :level: debug
 
+# It is also possible to override settings per environment:
+#:logging:
+#  :test:
+#    :pattern: "[%c|%.1l] %m\n"
```

Comment:
This is quite un-understandable in my POV... would it make sense to instead add an option to the loggers to only be environment specific? something line 
```yml
:loggers:
  :app:
    :enabled: true #this would always log
  :ldap:
    :enabled: [production, test] #a list of more then one environment
  :sql:
    :enabled: development #a single environment
```
This would be slightly similar to what we have for the smart proxy where enabled option can take `true`,`false`,`http`,`https`